Core/Systems/EntityModifier.cs

```diff
@@ -51,19 +53,20 @@ public void ApplyTo(Player player)
 
 		player.lifeRegen = (int)LifeRegen.ApplyTo(player.lifeRegen); // dont know if this is the right value
 		player.statManaMax2 = (int)MaximumMana.ApplyTo(player.statManaMax2);
-		// player.statManaMax - for when the overlay gets implimented
+		// player.statManaMax - for when the overlay gets implemented
 
 		player.manaRegen = (int)ManaRegen.ApplyTo(player.manaRegen); // dont know if this is the right value
-		player.statDefense += (int)Defense.ApplyTo((int)player.statDefense) - (int)player.statDefense;
-		
+		player.statDefense += (int)Defense.ApplyTo((int)player.statDefense) - player.statDefense;
+
 		player.endurance *= (int)DamageReduction.ApplyTo(player.endurance); // i think this is right..?
 		player.moveSpeed = MovementSpeed.ApplyTo(player.moveSpeed);
 		player.GetDamage(DamageClass.Generic) = player.GetDamage(DamageClass.Generic).CombineWith(Damage);
 
 		player.GetAttackSpeed(DamageClass.Generic) = Attackspeed.ApplyTo(player.GetAttackSpeed(DamageClass.Generic));
-		
+
 		player.GetKnockback(DamageClass.Generic) = player.GetKnockback(DamageClass.Generic).CombineWith(Knockback);
-		player.GetArmorPenetration(DamageClass.Generic) = Knockback.ApplyTo(player.GetArmorPenetration(DamageClass.Generic));
+		player.GetArmorPenetration(DamageClass.Generic) =
+			Knockback.ApplyTo(player.GetArmorPenetration(DamageClass.Generic));
```

Comment:
Don't think this should be Knockback.ApplyTo; though i'm probably the one who wrote it...
(though there are probably a lot of other issues with this method, no clue how you are supposed to increase these stats / what the intended way is for all of them)